src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.ts

```diff
@@ -19,16 +19,18 @@ export interface IQuickDates {
     hours: number;
 }
 
+export interface IRelevantEventsConfig {
+    eventType: string;
+    propertyMappings: [string, string][];
```

Comment:
this would probably be easier to interpret if it was something like

propertyMappings: {sourceProperty: string, targetProperty: string}[]
this makes it easier to expand later down the line.